.travis.yml

```diff
@@ -39,5 +39,9 @@ script:
     - sed -i "s#^test.mongo.exe.*#test.mongo.exe=$MONGOD#" test.cfg
     - sed -i "s#^test.mongo.wired_tiger.*#test.mongo.wired_tiger=$WIRED_TIGER#" test.cfg
     - cat test.cfg
-    - ant $ANT_TEST
+    - ant test-report
```

Comment:
This needs to use the ANT_TEST variable. We shouldn't run the same tests over and over and slow down the travis build just to get coverage.